app/views/admin/assets/index.html.erb

```diff
@@ -38,7 +38,7 @@
             <div class="button_column">
               <%= button_to "Mark As Spam", spam_admin_asset_path(asset.id), method: :put %>
               <%= button_to "Spam All for IP", mark_group_as_spam_admin_assets_path, method: :put, params: { mark_spam_by: { user_id: asset.user_id } } %>
-              <%= button_to "Delete User", admin_user_path(asset.user_id), data: { confirm: 'This will delete user forevAr. Are you sure?'}, method: :delete %>
+              <%= button_to "Delete User", delete_admin_user_path(asset.user&.login), data: { confirm: 'This will delete user forevAr. Are you sure?'}, method: :put %>
```

Comment:
Don't hate, but i think this button is here because of a copy/paste situation — we actually don't want to delete a user from the admin/assets page. We want to delete the asset!

Reply:
Hehe, np at all.
I _THINK_ we did put it there on purpose thinking that we want a short cut to delete a spammy user if we see it uploads a bunch of spammy assets.
My original desire was to just delete this button completely, which it seems like we might as well just do.